hokusai/staging.yml

```diff
@@ -81,14 +105,14 @@ metadata:
     service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: '300'
 spec:
   ports:
-    - port: 443
-      protocol: TCP
-      name: https
-      targetPort: 5000
     - port: 80
       protocol: TCP
       name: http
-      targetPort: 5000
```

Comment:
Ah so it may be a good idea to roll this out incrementally…. i.e. get the Nginx containers deployed before updating these listeners to route traffic through it.  Will roll this out so merge at will @zephraph